reminders/tests.py

```diff
@@ -1,20 +1,25 @@
 import pytest
-from django.utils import timezone
 from . import models
+from django.core import mail
 from datetime import datetime
-from events.models import EventParticipant
+from django.conf import settings
+from django.utils import timezone
 from events.tests import new_event, user0  # noqa: F401
+from events.models import EventParticipant
+from django.db.utils import IntegrityError
+from .models import Reminder, ReminderType
+from main.utilities import send_reminder_email
 from events.tests import event_participant_creator as participant0  # noqa: F401
-from django.core.exceptions import ValidationError
-from .models import Reminder, time_format
 
-METHOD_0 = models.ReminderType.EMAIL
+
 MESSAGES_0 = 'This is a test text.'
+METHOD_0 = models.ReminderType.EMAIL
 TIME_0 = datetime(2023, 3, 24, 12, 12, 12, 0, tzinfo=timezone.utc)
 
 JOIN_MEETING = 'Joined Meeting in {minutes} minutes'
```

Comment:
Maybe I have missed it while reading but have you used this constant later on?

Reply:
I have used it in the test_invalid_reminder_exist_twice function as stated in the comment above